src/core/callstack.c

```diff
@@ -503,19 +503,22 @@ void MVM_callstack_continuation_append(MVMThreadContext *tc, MVMCallStackRegion
 
 /* Walk the frames in the region, looking for the first bytecode one. */
 MVMFrame * MVM_callstack_first_frame_in_region(MVMThreadContext *tc, MVMCallStackRegion *region) {
```

Comment:
Hm, this name no longer describes what it does. I'm also a bit concerned about whether this can generate false positives now; for example, I think continuation handling uses this and relies upon the frame it finds really being in the region, otherwise it could find a root frame that isn't going to be within the continuation.

Reply:
While my comment on the name still kinda applies (maybe `first_frame_from_region` might be better), I I no longer think there's an actual bug here; I somehow misread the `->next` as if it were `->prev`.